sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdConnectionEvent.java

```diff
@@ -0,0 +1,10 @@
+//------------------------------------------------------------
```

Comment:
This header looks different from our other headers.

Reply:
Thanks, changed to be same as other files.